### PR TITLE
OD-535 [Feat] Added Logout option for link provider

### PR DESCRIPTION
**Pull request by OleksandrLavrynenko**
@sofiiakvasnevska @inna-bieshulia

OD-535https://weboo.atlassian.net/browse/OD-535

## Description
The "Log Out" option has been added to the "Link" component. It allows user to setup options for the "Log Out" components such as Button, Inline text link, Menu and List (no thumbnail)

## Screenshots/screencasts
https://storyxpress.co/video/kt1hvxxdlj0ten883

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@upplabs-alex-levchenko

### CODE REVIEW

File: js/interface.js [hunk 1]
```diff
@@ -145,25 +154,29 @@ Object.keys(btnSelector).forEach(function(key) {
...
-  $('#' + selectedValue + 'Section').addClass('show');
+/* Show/hide toggle function for sections on the same level.
+This is important for cases when we have a dropdown with additional sections on the inner levels (i.e logout) */
+function showSection(sectionDataKey, selectId) {
+  optionsValues[selectId].forEach(function(key) {
+    $sections[key] && $sections[key].toggleClass('show', key === sectionDataKey);
+  });
+}
 
+function onChange() {
```
Comment (tonytlwu) [inline js/interface.js]:
```suggestion
function onActionChange() {
```

**Review by tonytlwu — CHANGES_REQUESTED** [review]

File: js/interface.js (hunk 1 above)
Comment (OleksandrLavrynenko) [reply js/interface.js]: I fixed this

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska @inna-bieshulia This is ready for testing

Risk: High
Affected: All components features that support configuring links
Verify: Link actions are correctly displayed, loaded and saved

Squash and merge: Use PR title

File: interface.html [hunk 2]
```diff
@@ -33,22 +33,40 @@
               <option value="about-overlay">Open the about this app overlay</option>
               <option value="app">Link to an external application</option>
               <option value="exit-app">Return to app list</option>
+              <option value="logout">Log Out</option>
```
Comment (squallstar) [inline interface.html:36]: This should be "Log out", not "Log Out" as per screenshots in ticket. CS confirmed they want this as "Log out" during Sprint demo.

Comment (squallstar) [reply interface.html:36]: @tonytlwu cc ^

**Pull request merged**